.vscode/extensions.json

```diff
@@ -1,9 +1,33 @@
 {
   "recommendations": [
     "angular.ng-template",
-    "azurite.azurite",
+    "azurite.azurite", 
     "ms-azuretools.vscode-azurefunctions",
     "ms-azuretools.vscode-azurestorage",
-    "ms-dotnettools.csdevkit"
+    "ms-dotnettools.csdevkit",
+    
+    "github.copilot",
+    "github.copilot-chat",
+    
+    "ms-vscode.vscode-typescript-next",
```

Comment:
exclude this everywhere if this only adds preview builds that are not adding much